.github/workflows/ci.yml

```diff
@@ -10,6 +10,10 @@ env:
   DOCKER_TAG: buildenv
   DOCKER_FILE_PATH: ./docker-build-env/Dockerfile
 
+  TOKEN_LABEL: token-label
+  USER_PIN: 1234
+  SO_PIN: 4321
```

Comment:
Can we keep these the same just to avoid any problems this might cause? For example, libp11 doesn't have different engine control commands for PIN and SO_PIN, only `PIN: Specifies the pin code`.